data-prepper-core/src/main/java/com/amazon/dataprepper/peerforwarder/PeerForwarderAppConfig.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.peerforwarder;
+
+import com.amazon.dataprepper.parser.model.DataPrepperConfiguration;
+import org.springframework.context.annotation.Bean;
+
+public class PeerForwarderAppConfig {
+
+    @Bean
+    public PeerForwarderConfiguration peerForwarderConfiguration(final DataPrepperConfiguration dataPrepperConfiguration) {
+        if (dataPrepperConfiguration != null)
+            return dataPrepperConfiguration.getPeerForwarderConfiguration();
```

Comment:
Is PeerForwarderConfiguration nullable in DataPrepperConfiguration? If yes we need to handle that as well.

Reply:
I think PeerForwarderConfiguration can be nullable.
I can make this Bean return `Optional<PeerForwarderConfiguration>` and we won't start PeerForwarder if Optional.isPresent() is false.
```
@Bean
public Optional<PeerForwarderConfiguration> peerForwarderConfiguration(final DataPrepperConfiguration dataPrepperConfiguration) {
    return Optional.ofNullable(dataPrepperConfiguration.getPeerForwarderConfiguration());
}
```